pfio/MultiGroupServer.F90

```diff
@@ -799,10 +803,17 @@ subroutine start_back_writers(rc)
             end select
             select type (q=>msg)
             class is (AbstractDataMessage)
+               filename =q%file_name
+               call file_timer%start()
                call thread_ptr%put_dataToFile(q, address, _RC)
+               call file_timer%stop()
             end select
             call msg_iter%next()
          enddo
+         time = file_timer%get_total()
+         write(*,"(A, F9.6, A)"), " It takes ", time, "s to write file "//filename
```

Comment:
Query for @weiyuan-jiang or @tclune: At this point in pFIO, do we know the *size* of the file being written or could we estimate it? We could put in a MB/s number here as well.

Reply:
@weiyuan-jiang we really need this in terms of GB/sec.   Absolute time is good as well, so do both.   I suggest a format like the following to make things line up:
```
<sec> <GB/s>  wrote var <var> to file <file> on node <node-rank>
```

Reply:
The speed is not quite accurate. In one of my c24 experiment, with deflation level 1,  8.8 M file becomes 5.7 M.  Also, the size if not accurate. With 9M data ( excluding meta data) , the size of the file is 8.8M without deflation.  For small file, I think the speed is way off.  Output example:

Writing test_writing_prof.geosgcm_budi.20000415_1200z.nc4(size     0.094464 Mb) on node 0
time :     0.005555 s    test_writing_prof.geosgcm_budi.20000415_1200z.nc4
speed:    17.004751 Mb/s test_writing_prof.geosgcm_budi.20000415_1200z.nc4
Writing test_writing_prof.geosgcm_prog.20000415_1200z.nc4(size     9.088511 Mb) on node 0
time :     0.013254 s    test_writing_prof.geosgcm_prog.20000415_1200z.nc4
speed:   685.742554 Mb/s test_writing_prof.geosgcm_prog.20000415_1200z.nc4
Writing test_writing_prof.geosgcm_surf.20000415_1030z.nc4(size     2.860416 Mb) on node 0
time :     0.068756 s    test_writing_prof.geosgcm_surf.20000415_1030z.nc4
speed:    41.602604 Mb/s test_writing_prof.geosgcm_surf.20000415_1030z.nc4

Reply:
@weiyuan-jiang You probably want `MB` instead of `Mb`...I think.

Reply:
I think the raw data size for the variable is sufficient for our purposes.    Don't worry about what compression does on the far end.